src/store/useWaitlistStore.ts

```
@@ -0,0 +1,242 @@
+    get().recalculateStats(eventId);
+  },
+
+  addEntry: (eventId, entry) => {
+    set((state) => {
+      const existing = state.entries[eventId] || [];
+      const updated = sortByPosition([...existing, entry]);
+      return {
+        entries: { ...state.entries, [eventId]: updated },
+        userEntries: {
+          ...state.userEntries,
+          [eventId]: entry.user_id === entry.user_id ? entry : state.userEntries[eventId],
```

Biome (2.5.7)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)